src/hotspot/os/linux/globals_linux.hpp

```diff
@@ -81,6 +81,12 @@
   product(bool, UseCpuAllocPath, false, DIAGNOSTIC,                     \
           "Use CPU_ALLOC code path in os::active_processor_count ")     \
                                                                         \
+  /* SapMachine 2021-09-01: malloc-trace */                             \
+  product(bool, EnableMallocTrace, false, DIAGNOSTIC,                   \
+          "Enable malloc trace at VM initialization")                   \
+  product(bool, PrintMallocTraceAtExit, false, DIAGNOSTIC,              \
+          "Print Malloc Trace upon VM exit")                            \
+																	                                      \
```

Comment:
I think this line misses a '\' - not sure if it is just to match up local style or if there's some issue in the macro then.